src/package/abi/Microsoft.Windows.AbiWinRT.targets

```diff
@@ -46,7 +46,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     </PropertyGroup>
 
-    <!-- Using both cpp/winrt and abi/winrt together is unsupported. -->
+    <!-- Using both c++/winrt and abi/winrt together is unsupported. -->
     <Target Name="AbiWinRTValidateNoCppWinRT">
         <Error Condition="'$(CppWinRTPackage)' != ''" Text="Abi/WinRT and Cpp/WinRT create competing projections and are not compatible.
```

Comment:
and this

Reply:
🎵 Scott's got, the eye of the tiger... 🎵